tensorrt_llm/_torch/modules/mamba/causal_conv1d_triton.py

```
@@ -0,0 +1,1150 @@
+    x: torch.Tensor,
+    weight: torch.Tensor,
+    bias: Union[torch.Tensor, None],
+    conv_states: torch.Tensor,
+    query_start_loc: torch.Tensor,
+    seq_lens_cpu: List[int],
+    cache_indices: Optional[torch.Tensor] = None,
+    has_initial_state: Optional[torch.Tensor] = None,
+    activation: Optional[str] = "silu",
+    pad_slot_id: int = PAD_SLOT_ID,
+    validate_data=False,
+    **kwargs,
```

Ruff (0.14.11)

Unused function argument: `kwargs`

(ARG001)